src/server/svr_func.c

```diff
@@ -5141,6 +5174,8 @@ check_and_run_jobs(struct prov_vnode_info * prov_vnode_info)
 		DBPRT(("%s: Jobid: %s, startjob returned: %d\n",
 			id, pjob->ji_qs.ji_jobid, rc))
 
+	} else if (rc == -5) {
```

Comment:
rc is getting modified at line 5162  if ((rc = svr_startjob(pjob, 0))!=0) , will it not effect here.